ui/src/dashboards/components/dashboard_index/DashboardsIndex.tsx

```diff
@@ -87,35 +82,38 @@ class DashboardIndex extends PureComponent<Props, State> {
             </Page.Header.Left>
             <Page.Header.Right>
               <AddResourceDropdown
-                onSelectNew={this.handleCreateDashboard}
+                onSelectNew={createDashboard}
                 onSelectImport={this.summonImportOverlay}
                 onSelectTemplate={this.summonImportFromTemplateOverlay}
                 resourceName="Dashboard"
                 canImportFromTemplate={true}
+                status={this.addResourceStatus}
               />
             </Page.Header.Right>
           </Page.Header>
           <Page.Contents fullWidth={false} scrollable={true}>
             <div className="col-md-12">
               <GetResources resource={ResourceTypes.Dashboards}>
-                <DashboardsIndexContents
-                  filterComponent={() => (
-                    <SearchWidget
-                      placeholderText="Filter dashboards..."
-                      onSearch={this.handleFilterDashboards}
+                <GetResources resource={ResourceTypes.Labels}>
+                  <GetAssetLimits>
+                    <DashboardsIndexContents
```

Comment:
I'm worried that this new component + redux pattern we have for fetching resources will discourage us from fetching resources in parallel, rather than in serial. In this example it looks like the dashboards, labels, and asset limits could all be fetched at the same time.

I don't have a sense of what the user impact of this is (does it make loading the dashboards index much slower?) Also not something to fix in this PR, just thinkin about it.

Reply:
I actually moved these all the fetchers to one component one after another to highlight that it was silly that we were doing them serially. I'll make an issue to consider ways to parallelize/ GetResources could probably take an array and wait until all promises are resolved.

Reply:
Awesome. It might also be worth just writing specialized fetcher components whenever we need them, rather than trying to make `GetResources` a generic super-flexible component.